Chipyard Integration (#56)
ucb-bar/riscv-sodor: README.md

About The Sodor Processor Collection
====================================

> **Note: This repo has been updated to be a part of [Chipyard](https://github.com/ucb-bar/chipyard) Soc Generator.**

Reviewer: ```suggestion
**Note: This repo has been updated to be used with the [Chipyard](https://github.com/ucb-bar/chipyard) SoC Generator.**
```

> **For the old version of Sodor (which is no longer maintained), see https://github.com/ucb-bar/riscv-sodor/tree/sodor-old.**

Reviewer: ```suggestion
**For the old self-contained version of Sodor (which is no longer maintained), see https://github.com/ucb-bar/riscv-sodor/tree/sodor-old.**
```

All processors talk to a simple scratchpad memory (asynchronous,
single-cycle), with no backing outer memory (the 3-stage is the exception

> \- its scratchpad is synchronous). Programs are loaded in via the Chipyard debugger, effectively making the

Reviewer: ```suggestion
\- its scratchpad is synchronous). Programs are loaded in via JTAG or TSI,
```

Getting the repo and Building the processor emulators
=====================================================

This repo is **NOT** a self-running repository. Please follow the instruction in 

> https://chipyard.readthedocs.io/en/latest/ to set up Chipyard and run the test. 

Reviewer: ```suggestion
https://chipyard.readthedocs.io/en/latest/ to set up Chipyard and simulate Sodor cores.
```
